ts/Series/OHLC/OHLCPoint.ts

```diff
@@ -33,36 +34,22 @@ const {
  *
  * */
 
-class OHLCPoint extends ColumnSeries.prototype.pointClass {
+class OHLCPoint extends HLCSeries.prototype.pointClass {
```

Comment:
Shouldn't be simply `HLCPoint`? (Then the import won't be unnecessary)

Reply:
A simple import results in bigger packages. `SeriesRegistry` deconstruction and `pointClass` er the byte-saving way to reference an "external" series.